### ISSUE TITLE
Participated filter doesn't seem to be working

### ISSUE THREAD

**Issue by jeffreyk** [ptsochantaris/trailer#355]
I have setup a watched repository with a participated filter for pull requests but I am not seeing pull requests that show me as a participant on github. I have no filter on author based hiding. Maybe I am doing something wrong? Currently to see those pull requests I have to change the filter to all.

**Comment by ptsochantaris** [comment]
Hi @jeffreyk - The "Participated" section will show items which you have added comments to by default. There are also options to add items there which, for example, mention you or your team, have review requests, etc. These are all controlled from preferences. This section isn't related to what GitHub may consider "participated" - I'm not sure what criteria may apply there. If you think that the logic in "Participated" should match that definition more closely, I'm happy to discuss! (But let me know what you think is the difference/divergence.)

I hope I have understood your question correctly, let me know if I haven't.

**Comment by jeffreyk** [comment]
Thanks for the quick response! 

I think the way github defines participated is if I made a commit. I don't know about other people, but I think it would be nice for trailer to more closely align with the github definition of participated.

Though ultimately I would like to be able to follow only pull requests that I opened or paired on. Is adding a comment and setting the filter to participated the only way to do this?
